scans/auth/plans_and_scripts/dev_bba.yaml

```diff
@@ -0,0 +1,48 @@
+env:
+  contexts:
+  - name: BBA_Test
+    urls:
+    - http://localhost:9091/auth/simple-json-cookie/
+    includePaths:
+    - http://localhost:9091/auth/simple-json-cookie/.*
+    authentication:
+      method: browser
+      parameters:
+        loginPageUrl: http://localhost:9091/auth/simple-json-cookie/
```

Comment:
Ideally we'll have one common plan for "clean" bba, with all of the variables parameterized.

Reply:
Ideally